packages/browser.smash

```diff
@@ -150,6 +150,12 @@
         }, await $(element));
     }
 
+    * Scroll to top {
+        await executeScript(function() {
+            window.scrollTo(0, 0);
+        });
+    }
```

Comment:
This is good. It only need a unit test with a manual verify at the end. See `tests/packages/browser.smash` with `#manual` tests as an example.